pom.xml

```diff
@@ -2049,6 +2049,14 @@
         </dependencies>
     </dependencyManagement>
 
+    <repositories>
```

Comment:
Please remove this repository

Reply:
yes that will be removed once 1.4.1 is officially released